tests/cpp_compat_tests.rs

```diff
@@ -0,0 +1,46 @@
+#![cfg(all(feature = "rust", feature = "cpp"))]
+
+use fastpfor::cpp::Codec32;
+use fastpfor::rust::Integer;
```

Comment:
If these are only used as traits, you may want to do this:

```suggestion
use fastpfor::cpp::Codec32 as _;
use fastpfor::rust::Integer as _;
```

Reply:
that's what i'm here for :D